admin_queries/workspaces.sql

```diff
@@ -0,0 +1,49 @@
+WITH
+assessment_users_with_workspace_counts AS (
+    SELECT
+        a.id AS assessment_id,
+        v.authn_user_id,
+        count(*) AS workspace_count
+    FROM
+        workspaces AS w
+        JOIN variants AS v ON (v.workspace_id = w.id)
+        JOIN instance_questions AS iq ON (iq.id = v.instance_question_id)
+        JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
+        JOIN assessments AS a ON (a.id = ai.assessment_id)
+    WHERE w.state != 'stopped'
```

Comment:
Will this query grab recently stopped workspaces as well?

Reply:
No, but I think that's ok because we aren't displaying any details about individual workspaces. This query is more to find out which courses/assessments are causing all the workspace load (equivalent to `current_submission_activity.sql`). I find this useful after looking out CloudWatch and thinking "wow, there is huge load spike, what's causing that". Then this query can tell me "it's CS 357, they just started MP2".

Does that sound reasonable, or do you think we should include recently-stopped workspaces in this?

However, your question did make me realize that the above query was including `uninitialized` workspaces, which is wrong. I fixed it just now.

Reply:
Right that seems reasonable, I only ask because the query description includes recently stopped workspaces.  I'm fine with not including recently stopped workspaces.

Reply:
Ooops. Good call. Description is fixed.